docs/oauth2/oidc/id_tokens.rst

```diff
@@ -38,12 +38,13 @@ You can switch to jwcrypto library if you want to return JWE instead.
 
         super().__init__(self, **kwargs)
 
-    def get_id_token(self, token, token_handler, request):
+    def fill_id_token(self, id_token, token, token_handler, request):
```

Comment:
Yeah, I struggled a bit before naming that one, I was a bit reluctent to use `finalize` because the implementor NEED to fill a couple of mandatory fields and not only do the signature. However, one of the other, maybe `finalize_` is better than `fill_` which does not include the "finalization" part. Let me think about it a bit more...